src/Tizen.Network.Stc/Tizen.Network.Stc/StcInfo.cs

```diff
@@ -0,0 +1,281 @@
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>Interface name.</value>
+        public string InterfaceName
+        {
+            get
+            {
+                string ifaceName;
+                int ret = Interop.Stc.Info.GetInterfaceName(_infoHandle, out ifaceName);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get interface name from info, Error - " + (StcError)ret);
+                    return string.Empty;
+                }
+                return ifaceName;
+            }
+        }
+
+        /// <summary>
+        /// A property to get "from" value(start) of time interval from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>from(start) of time interval.</value>
+        public DateTime From
+        {
+            get
+            {
+                DateTime from;
+                DateTime to;
+                int ret = Interop.Stc.Info.GetTimeInterval(_infoHandle, out from, out to);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get time interval(from value) from info, Error - " + (StcError)ret);
+                }
+                return from;
+            }
+        }
+
+        /// <summary>
+        /// A property to get "to" value(end) of time interval from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>to(end) of time interval.</value>
+        public DateTime To
+        {
+            get
+            {
+                DateTime from;
+                DateTime to;
+                int ret = Interop.Stc.Info.GetTimeInterval(_infoHandle, out from, out to);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get time interval(to value) from info, Error - " + (StcError)ret);
+                }
+                return to;
+            }
+        }
+
+        /// <summary>
+        /// To get the interface type from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <returns>Interface type.</returns>
+        /// <exception cref="InvalidOperationException">Thrown when method fails due to an invalid operation/value.</exception>
+        public NetworkInterface GetInterfaceType()
+        {
+            NativeNetworkInterface ifaceType;
+            int ret = Interop.Stc.Info.GetInterfaceType(_infoHandle, out ifaceType);
+            if (ret != (int)StcError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to get Interface type from info, Error - " + (StcError)ret);
+            }
+            if (ifaceType == NativeNetworkInterface.Unknown)
+            {
+                throw new InvalidOperationException("Interface Type is Unknown.");
```

Comment:
Is it possible scenario that throw exception?
Could you explain which case could be occurred error?

Acutally, I want to change it to getter only property without excpetion

Reply:
As suggested in the discussion: Exception throw is removed, Unknown is passed to app as 'Others' and made it a Property.